app/webpack/stylesheets/govuk-secondary-button.scss

```diff
@@ -1,24 +1,24 @@
 .govuk-button.govuk-secondary-button {
-  background-color: govuk-colour("light-grey");
   color: #0b0c0c;
+  background-color: govuk-colour("light-grey");
   box-shadow: 0 2px 0 #b5babe;
 }
 
-.govuk-button.govuk-secondary-button:active, 
-.govuk-button.govuk-secondary-button:hover, 
-.govuk-button.govuk-secondary-button:focus  {
-  background-color: #d0d3d6;
+.govuk-button.govuk-secondary-button:active,
```

Comment:
I'm unsure why we need these styles at all.

I'd expect us to be using `govuk-button--secondary` as per the design system, so we may be able to remove this stylesheet and replace the usage of `govuk-secondary-button` in the `_next_action_button` partial.

Reply:
I spotted a bit of this, therefore I create https://dsdmoj.atlassian.net/browse/AP-3699 to replace variables, functions and mixins.